### DISCUSSION TITLE
Is this type of bar chart achievable?

### DISCUSSION THREAD

**Discussion by aik117** [Abhinandan-Kushwaha/react-native-gifted-charts#918] [Q&A] [answered]
Hi @Abhinandan-Kushwaha wanted to know if below type of bar chart is achievable in this package?
![image](https://github.com/user-attachments/assets/69c81c37-97c2-4375-b796-e432dbdf8db2)
Please let us know. Thanks.

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Yes @aik117 it can be achieved using the `<PopulationPyramid>` component

**Reply by aik117** [reply]
Thanks. It would have been nice if we had the y-Axis on the right side as well.
